react/StoreContextProvider.js

```diff
@@ -102,68 +103,70 @@ class StoreContextProvider extends Component {
+                <meta name="keywords" content={metaTagKeywords} />
+                <meta name="copyright" content={storeName} />
+                <meta name="author" content={storeName} />
+                <meta name="country" content={country} />
+                <meta name="language" content={locale} />
+                <meta name="currency" content={currency} />
+                <meta name="robots" content={metaTagRobots || META_ROBOTS} />
+                <meta httpEquiv="Content-Type" content={CONTENT_TYPE} />
+                {canonicalPath && (
+                  <link
+                    rel="canonical"
+                    href={`https://${window.__hostname__ ||
+                      (window.location &&
+                        window.location.hostname)}${canonicalPath}`}
+                  />
+                )}
+              </Helmet>
+              {/* PWA */}
+              {hasManifest && (
+                <Helmet>
+                  <meta name="theme-color" content={manifest.theme_color} />
+                  <link rel="manifest" href="/pwa/manifest.json" />
+                  <script type="text/javascript" src="/pwa/workers/register.js" />
+                  {hasManifest &&
+                    manifest.icons &&
+                    manifest.icons
+                      .filter(({ sizes }) => iOSIconSizes.includes(sizes))
+                      .map(icon => (
+                        <link
+                          key={icon.src}
+                          rel="apple-touch-icon"
+                          sizes={icon.sizes}
+                          href={icon.src}
+                        />
+                      ))}
+                  <meta name="apple-mobile-web-app-capable" content="yes" />
+                  {splashes &&
+                    splashes.map(splash => (
                       <link
-                        key={icon.src}
-                        rel="apple-touch-icon"
-                        sizes={icon.sizes}
-                        href={icon.src}
+                        key={splash.src}
+                        href={splash.src}
+                        sizes={splash.sizes}
+                        rel="apple-touch-startup-image"
                       />
                     ))}
-                <meta name="apple-mobile-web-app-capable" content="yes" />
-                {splashes &&
-                  splashes.map(splash => (
-                    <link
-                      key={splash.src}
-                      href={splash.src}
-                      sizes={splash.sizes}
-                      rel="apple-touch-startup-image"
-                    />
-                  ))}
-              </Helmet>
-            )}
-            <OrderFormProvider>
-              <div className="vtex-store__template">{this.props.children}</div>
-            </OrderFormProvider>
-          </DataLayerProvider>
-        </PixelProvider>
+                </Helmet>
+              )}
+              <OrderFormProvider>
```

Comment:
Better to add it here next to `OrderFormProvider`, reduces the impact on the code, and they are more alike :smile: